ee/institution/admin/saml.sql

```diff
@@ -51,14 +88,28 @@ WITH deleted_authn_provider AS (
     institution_id,
     row_id,
     old_state
-  ) SELECT ...
-), audit_logs_deleted_saml_provider (
+  ) SELECT
+    $authn_user_id as authn_user_id,
+    'institution_authn_providers' as table_name,
+    'delete' as action,
+    $institution_id as institution_id,
+    dap.id as row_id,
+    to_jsonb(dap) AS old_state
+  FROM deleted_authn_provider AS dap
+), audit_logs_deleted_saml_provider AS (
   INSERT INTO audit_logs (
     authn_user_id,
     table_name,
     action,
     institution_id,
     row_id,
     old_state
-  ) SELECT ...
+  ) SELECT
+    $authn_user_id as authn_user_id,
+    'saml_providers' as table_name,
+    'delete' as action,
+    $institution_id as institution_id,
+    dsp.id as row_id,
+    (to_jsonb(dsp) || jsonb_build_object('private_key', 'REDACTED')) AS old_state
```

Comment:
Similar comment here about the `AS ...` names being unnecessary (but fine to leave if you prefer).

Reply:
Removed in eaa756d.